pkg/storage/unified/resource/resource.proto

```diff
@@ -192,7 +200,11 @@ message ReadRequest {
   ResourceKey key = 1;
 
   // Optionally pick an explicit resource version
-  int64 resource_version = 3;
+  int64 resource_version = 2;
```

Comment:
^^ does the number count as a breaking change?

Reply:
Technically Yes. That being said, this will only break getting and older version of a resource and I don't think we have any consumer yet 🤔 . I am wondering if the server will reject a request from a older version of the client (given it receives 3 fields instead of 2 ?)